src/library/models.py

```diff
@@ -744,7 +744,14 @@ def get_reading_data(cls, period: Period, days=0, sort='name', books_sort='title
                 if books_sort == ReadingDetailsSort.TIME:
                     reading_data_for_one_user['books'].sort(reverse=True, key=lambda item: item['hours'])
                 if books_sort == ReadingDetailsSort.LASTVIEW:
-                    reading_data_for_one_user['books'].sort(reverse=True, key=lambda item: item['last_view'])
+                    book_list = reading_data_for_one_user['books']
+                    last_view_null = []
+                    for abook in book_list:
+                        if abook['last_view'] is None:
+                            last_view_null.append(abook)
+                            book_list.remove(abook)
+                    book_list.sort(reverse=True, key=lambda item: item['last_view'])
+                    book_list.extend(last_view_null)
```

Comment:
Sure, that works, or you could do something like `sort(key=lambda item: item['last_view'] or a_very_old_date)`

Reply:
That's similar to an approach I abandoned.  During the earlier creation of the map, from which the `book_list` is derived, set the `last_view` property to a year ago if the database record's value was null.  Then, the sort would just proceed as before, putting all the items from the fake year ago at the end.  The problem was that you could not distinguish "a year ago" from a legitimate date.  If we had a constant set to `VERY_OLD_DATE`, that could act as a marker.  I'll think some more on that.